### PR TITLE
Add splash screen

### PR DESCRIPTION
**Pull request by scottliv**

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -59,7 +60,7 @@ export default class App extends Component {
 
   render() {
```
Comment (siddharthparmar7) [inline js/index.js:61]: this.state -> loadingUser should come from the redux store, it should not be in the component state. Please make the change.

Comment (siddharthparmar7) [reply js/index.js:61]: just use the userLoading(true/false).

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Please make the changes and I will approve this pull request. Thank you!

File: js/index.js (hunk 1 above)
Comment (scottliv) [reply js/index.js:61]: Mackenzie and I tried to resolve this but could not make it work.

**Review by siddharthparmar7 — APPROVED** [review]
Thank you!

**Pull request merged**